syntax = "proto3";

message secret {
  //the following values correspond to
  // - protobuf data type
  // - name
  // - numeric tag corresponding to the ordering in which the fields are serialized
   repeated string cipherText = 1;
   repeated string exponent = 2;

}

message globalDist {
  repeated string cipherText = 1;
  repeated string exponent = 2;
}

service safeTransmission {
  rpc exchangeData (secret) returns (globalDist);
}